test/cpp/lite_interpreter/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+set(LITE_INTERPRETER_TEST_DIR "${TORCH_ROOT}/test/cpp/lite_interpreter")
```

Comment:
There are test_lite_interpreter.cpp and test_lite_trainer.cpp in jit folder, to test the end-to-end flow. It would be great to rename this folder to "lite_interpreter_runtime", or "mobile_runtime" to avoid confusion.

Reply:
Updated with the name `lite_interpreter_runtime`, because people are more familiar with this name. Can consider to rename it to mobile_runtime later if a shorter name is preferred.